osmaxx/core/templates/pages/about_us.html

```diff
@@ -0,0 +1,28 @@
+{% extends "osmaxx/base.html" %}
+{% load i18n %}
+
+{% block main %}
+<div class="container-row content">
+    <h2>About OSMaxx</h2>
+    <p>
+        OSMaxx (short for "OpenStreetMap arbitrary excerpt export") cuts out OpenStreetMap data, processes it to
+        geodata, converts it to typical GIS fileformats and offers those for download.
+    </p>
+    <h3>Useful links</h3>
+    <p>Can be found in our separate <a href="{% url 'pages:downloads' %}">downloads page</a></p>
+
+    <h3>Impressum</h3>
+    <p>
+        OSMaxx has been developed by the Geometalab at Hochschule Zürich.
+        Geometalab can be reached via email: <a href="mailto:geometalab@hsr.ch">geometalab@hsr.ch</a>.
+    </p><p>
+        <strong>Geometalab</strong><br />
+        HSR Hochschule für Technik Rapperswil<br />
+        Institute for Software<br />
+        c/o Stefan Keller<br />
+        Oberseestrasse 10<br />
+        Postfach 1475<br />
+        CH-8640 Rapperswil<br />
```

Comment:
Do we need the postal address here? (I know it's required in Germany, but Swiss publication laws are more liberal AFAIK.) More useful for interested users would probably be links to geometalab, IFS and HSR websites. (On the IFS website they could still find our postal address if desired.)